configuration/jsr223.md

```diff
@@ -290,7 +290,7 @@ The `default` preset is preloaded, so it does not require importing.
 | `things`                | Instance of `org.openhab.core.thing.ThingRegistry`                                       |
 | `rules`                 | Instance of `org.openhab.core.automation.RuleRegistry`                                   |
 | `events`                | (internal) Used to send events, post commands, etc. [Details](#events-operations) below] |
-| `actions`               | Instance of `org.openhab.core.thing.binding.ThingActions`                                |
+| `actions`               | Instance of `org.openhab.core.automation.module.script.defaultscope.ScriptThingActions`  |
```

Comment:
I don’t think that is correct.

For example do the `AstroActions` implement `org.openhab.core.thing.binding.ThingActions`: 
https://github.com/openhab/openhab-addons/blob/b316096e2992f55acca235257dc07d2ef9f49f42/bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/action/AstroActions.java#L47

Reply:
As far as I can understand the doc, on the left-hand side of the table are context variables of rules scripts and imported constants, and on the right-hand side are their respective types.

If that's correct, then `actions` is a context variable of a rule script.  
If I understand correctly, a value for this variable is set there `org.openhab.core.automation.module.script.internal.defaultscope.DefaultScriptScopeProvider#DefaultScriptScopeProvider` as the field `thingActions` of type `org.openhab.core.automation.module.script.internal.defaultscope.ScriptThingActionsImpl`, which implements `org.openhab.core.automation.module.script.defaultscope.ScriptThingActions`, that I've specified in the PR.

Talking about `org.openhab.core.thing.binding.ThingActions` like `AstroActions`, they can be obtained by calling `org.openhab.core.automation.module.script.defaultscope.ScriptThingActions#get` on the context variable `actions`. But `org.openhab.core.thing.binding.ThingActions` is not the type of the context variable itself.

Reply:
I have had a closer look and you are right.
Anway: Instead of just correcting the description, I would prefer that the `get` method returns a Thing action.

Reply:
:thinking: 

I’m not quite understand. Could you please elaborate on the idea a bit more?